README.md

```diff
@@ -40,103 +83,65 @@ To run a specific unittest:
-requests, or update the VA configuration's `portConfig` to use ports 80 and 443
-to match how the VA operates in production and staging environments. If you use
-a host-based firewall (e.g. `ufw` or `iptables`) make sure you allow connections
-from the Docker instance to your host on the required ports.
+Boulder's default VA configuration (`test/config/va.json`) is configured to connect to port 5002 to validate HTTP-01 challenges and port 5001 to validate TLS-SNI-01 challenges. If you want to solve challenges with a client running on your host you should make sure it uses these ports to respond to validation requests, or update the VA configuration's `portConfig` to use ports 80 and 443 to match how the VA operates in production and staging environments. If you use a host-based firewall (e.g. `ufw` or `iptables`) make sure you allow connections from the Docker instance to your host on the required ports.
 
-If a base image changes (i.e. `letsencrypt/boulder-tools`) you will need to rebuild
-images for both the boulder and bhsm containers and re-create them. The quickest way
-to do this is with this command:
+If a base image changes (i.e. `letsencrypt/boulder-tools`) you will need to rebuild images for both the boulder and bhsm containers and re-create them. The quickest way to do this is with this command:
 
     ./docker-rebuild.sh
 
-Slow start
-----------
+### Without Docker
 
-If you can't use the Docker setup, here are instructions for setting up a
-Boulder development environment without it.
+If you can't use the Docker setup, here are instructions for setting up a Boulder development environment without it.
 
-We recommend setting git's [fsckObjects
-setting](https://groups.google.com/forum/#!topic/binary-transparency/f-BI4o8HZW0/discussion)
-for better integrity guarantees when getting updates.
-
-Boulder requires an installation of libtool-ltdl, goose, SoftHSM, and MariaDB 10.1 to work correctly. If you want to save some trouble installing MariaDB and SoftHSM you can run them using Docker:
+Boulder requires an installation of libtool-ltdl, goose, SoftHSMv2, and MariaDB 10.1 to work correctly. If you want to save some trouble installing MariaDB and SoftHSM you can run them using Docker:
 
     docker-compose up -d bmysql bhsm
 
-Also, Boulder requires Go 1.9 or above. This version may not be
-available in OS repositories. If so, you will have to install from https://golang.org/dl/.
-Add ```${GOPATH}/bin``` to your path.
+Also, Boulder requires Go 1.9 or above. This version may not be available in OS repositories. If so, you will have to install from https://golang.org/dl/. Add ```${GOPATH}/bin``` to your path.
 
 Ubuntu:
 
-    sudo apt-get install libltdl3-dev mariadb-server
+    sudo apt-get install libltdl3-dev mariadb-server softhsm2
 
 CentOS:
 
-    sudo yum install libtool-ltdl-devel MariaDB-server MariaDB-client
+    sudo yum install libtool-ltdl-devel MariaDB-server MariaDB-client softhsm
```

Comment:
Have you confirmed that CentOS, Arch, and OS X will install v2 when plain `softhsm` is requested?

Reply:
macOS is v2
>brew info softhsm
`softhsm: stable 2.3.0 (bottled)`